giotto/images/preprocessing.py

```diff
@@ -0,0 +1,123 @@
+"""Image preprocessing."""
+# License: GNU AGPLv3
+
+import numbers
+import numpy as np
+from sklearn.base import BaseEstimator, TransformerMixin
+from joblib import Parallel, delayed, effective_n_jobs
+from sklearn.utils import gen_even_slices
+from sklearn.utils.validation import check_is_fitted, check_array
+from ..utils.validation import validate_params
+
+
+@adapt_fit_transform_docs
```

Comment:
you also need to import the wrapper from `giotto/utils/_docs.py` (this is why the pipelines failed). sorry i was not precise enough the first time